gst/nnstreamer/include/nnstreamer_cppplugin_api_filter.hh

```diff
@@ -174,7 +174,9 @@ class tensor_filter_subplugin
 
   virtual ~tensor_filter_subplugin (); /**< called by Close */
 
-  virtual void invoke (const GstTensorMemory *input, GstTensorMemory *output) = 0;
+  virtual void invoke (GstTensorFilterProperties *prop,
```

Comment:
This breaks backward compatibility and there are companies and Samsung's business divisions that have their own proprietary subplugins.

Because this is C++, not C, you may apply some techniques to avoid it while you add ```prop```. (e.g., operator override, default argument value, ...)

Reply:
The code has been restored.
Since tensor information is managed as a class attributes in the c++ subplugin, the guide has been modified to update the tensor information in invoke function for dynamic invoke.